lib/tools/src/tprof.erl

```diff
@@ -257,16 +257,16 @@ children started by the user-supplied function are kept, it is developer's
 responsibility to ensure cleanup.
 
 ```erlang
-      9> tprof:profile(timer, sleep, [100000], #{timeout => 1000}).
+9> tprof:profile(timer, sleep, [100000], #{timeout => 1000}).
```

Comment:
Mainly for my knowledge: is this `text` or `erlang` or some other kind of format? It's not a valid Erlang program (because it's typed in the shell and it has `9>` shell prompt).

Reply:
The Erlang syntax highlighting in ExDoc can deal with the prompt being there so that shell examples work nicely. However when you get big printouts such as some other examples in this code it becomes not very nice.